apps/atomy-q/API/app/Console/Commands/AiVerifyContractsCommand.php

```
@@ -0,0 +1,152 @@
+    protected $signature = 'atomy:ai-verify-contracts
+        {--endpoint-group=* : Restrict verification to one or more endpoint groups}
+        {--tenant-id=plan6-tenant : Tenant identifier used in sample payloads}
+        {--rfq-id=plan6-rfq : RFQ identifier used in sample payloads}';
+
+    protected $description = 'Run provider contract verification requests for every configured Atomy-Q AI endpoint group.';
+
+    public function __construct(
+        private readonly ProviderDocumentIntelligenceClient $documentClient,
+        private readonly ProviderNormalizationClient $normalizationClient,
+        private readonly ProviderSourcingRecommendationClient $recommendationClient,
+        private readonly ComparisonAwardAiClientInterface $comparisonAwardClient,
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $comparisonAwardClient. Keep variable name length under 20. (undefined)

(LongVariable)